src/hotspot/share/code/relocInfo.hpp

```diff
@@ -130,11 +130,7 @@ class nmethod;
 //   Instruction types: memory (load), set (load address)
 //   Data:  []       an oop stored in 4 bytes of instruction
 //          [n]      n is the index of an oop in the CodeBlob's oop pool
-//          [[N]n l] and l is a byte offset to be applied to the oop
-//          [Nn Ll]  both index and offset may be 32 bits if necessary
-//   Here is a special hack, used only by the old compiler:
-//          [[N]n 00] the value is the __address__ of the nth oop in the pool
-//   (Note that the offset allows optimal references to class variables.)
+//          [Nn]     index may be 32 bits if necessary
```

Comment:
Lines 132 and 133 could be combined into something like:
```
// [[N]n]  index of an oop in the CodeBlob's oop pool
```
which seems consistent with other descriptions.

Reply:
Okay.